data/json/vehicleparts/stowed_parts.json

```diff
@@ -0,0 +1,257 @@
+[
+  {
+    "type": "vehicle_part",
+    "id": "stowed_fridge",
+    "name": { "str": "tied-down household fridge" },
+    "categories": [ "other" ],
+    "color": "white",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A fridge, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "apartment_fridge",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "tool_rope_long", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "apartment_fridge" }, { "item": "rope_30" } ],
```

Comment:
are you sure disassembly will return your rope back again? because it looks like it would consume apartment_fridge and tool_rope_long, but will remove only apartment_fridge?

Reply:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/78005870/192ed324-5959-413e-b21e-03dd21e9696f)
Ah thanks for the catch, fixed now!

Reply:
It was not, in fact, answer to my question

Reply:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/78005870/77f53a62-3572-4da6-8ea0-e0c22e4814d1)
It still gives me the rope item